src/System.Net.Security/tests/FunctionalTests/SslStreamAlpnTests.cs

```diff
@@ -61,6 +61,62 @@ private bool DoHandshakeWithOptions(SslStream clientSslStream, SslStream serverS
         }
 
         [Fact]
+        public void SslStream_StreamToStream_ClientCancellation_Throws()
+        {
+            VirtualNetwork network = new VirtualNetwork();
+            using (var clientStream = new VirtualNetworkStream(network, false))
+            using (var serverStream = new VirtualNetworkStream(network, true))
```

Comment:
Nit: please name the bool args to VirtualNetworkStream; it's not clear at the call site what the false and true mean.